pkg/cmd/codespace/create.go

```diff
@@ -234,7 +236,7 @@ func getMachineName(ctx context.Context, apiClient apiClient, repoID int, machin
 	machineNames := make([]string, 0, len(machines))
 	machineByName := make(map[string]*api.Machine)
 	for _, m := range machines {
-		machineName := m.DisplayName
+		machineName := buildDisplayName(m.DisplayName, m.PrebuildAvailability, makeGrayText)
```

Comment:
Coloring the text output here will appear to work (gray will be used in the select menu), but I wouldn't suggest actually using it because the Survey library that we use does not have good support for ANSI colors in menus. Even though color will be displayed properly, re-rendering the prompt on narrow terminals might lead to breakages because Survey isn't able to reliably measure the width of text that was printed. I would suggest not using any coloring here.